src/gc/gchandletableimpl.h

```diff
@@ -17,6 +17,14 @@ class GCHandleTable : public IGCHandleTable
     virtual void* GetHandleTableContext(void* handleTable);
 
     virtual void* GetHandleTableForHandle(OBJECTHANDLE handle);
+
+    virtual OBJECTHANDLE CreateHandleOfType(void* table, Object* object, int type);
```

Comment:
Should the definition of `OBJECTHANDLE` be on the interface? I see it's defined in `vars.hpp` and `gcenv.base.h` today.